README.md

```diff
@@ -1,3 +1,11 @@
 # Silverstripe CMS roadmap and support timeline JSON
 
-Holds JSON data used for the Silverstripe CMS roadmap and support timeline
+Holds `data.json` used for the Silverstripe CMS roadmap and support timeline
+
+The other files in the repository are for validating that data.json is valid in ci
+
+## What to update when a new version of Silverstripe CMS is released
+
+As part of a Silverstripe CMS release, when there is certainty about the exact day the release is happening (which is probably on the actual day of the release), the `"releaseDate"` node of the release should be updated to be in `"YYYY-MM-DD"` format (NOT `"YYYY-MM"` format)
```

Comment:
```suggestion
As part of a Silverstripe CMS release, after the release has been tagged, the `"releaseDate"` node of the release should be updated to be in `"YYYY-MM-DD"` format (NOT `"YYYY-MM"` format)
```
Better to have the timeline update when the tag is actually available. While it's not _super_ likely someone will see it before the release is out, there _is_ a chance that if we updated it early someone would try to install a version that hasn't been released stable.

Reply:
Updated